docs/documentation/stories/rc.0-update.md

```diff
@@ -311,20 +311,23 @@ We've updated a lot of packages over the last months in order to keep projects u
 
 Additions or removals are found in bold below.
 
+`"angular-cli": {},` was **removed**
```

Comment:
@filipesilva What I'm referencing is an "angular-cli" object that the betas put in to the project.json that is outside of the "dependencies" and "devDependencies" objects.  In the betas, that line was previously between "license" and "scripts" and in the RC that item no longer exists, so I just included that note so people could clean it up.  
Since that extraneous object doesn't currently break anything and we don't want people to confuse it with the line you referenced, I'll take it out of this pull request and rebase as you requested so that the things that are broken can get fixed. Thanks.